src/Tween.ts

```diff
@@ -280,11 +281,11 @@ export class Tween {
         if (end.length > start.length) {
           // so in this case we will increase number of starting points
           trueStart = start;
-          start = Util._prepareArrayForTween(start, end, node.closed());
+          start = Util._prepareArrayForTween(start, end, (node as Line).closed());
```

Comment:
I'm not sure here. Is it a really logic about `line`? Mb I can do a type guard helper above.

Reply:
Probably we can just wite `node.getAttr('closed')` here.

Reply:
Are there any test which covers it? 
Can you tell me how can I test it?

Reply:
Related tests are here: https://github.com/konvajs/konva/blob/master/test/unit/Tween-test.js#L255